### ISSUE TITLE
Why BookStack's documentation is in Hugo?

### ISSUE THREAD

**Issue by marcbria** [BookStackApp/BookStack#5181]
Hi Dan (et. al.),

I'm evaluating BookStack as a tool for documenting other free software and I need to say that I like it a lot.

I mean, I find some limitations (see below) but my biggest doubt here is... **why don't you use BookStack to document BookStack?** The fact that you don't makes me think there are issues that I haven't accounted for.

Apart from this (just in case you can comment if I'm wrong or there is any alternative workaround) those are the main features I'm missing for this use case:

- Soft-versions:  We need to refer the soft version for each page we write. I though it could probably be achieved it with “tags” or better with a hack (as we like it to work as in readTheDocs... where switching the version changes the document)
- Multilang: Everything (shelve, book, chapter, page) could be translated to multiple langs. BookStack is supposed to be multilang but only at UI level, isn't it? I mean, there is no way to keep documents in multiple langs appart from tags.
- Autoscreenshot: Not by it's own but this could probably be implemented in BS with third party tools, isn't it?

Thanks for your help and for you time developing this wonderful software,
m.

**Comment by watschi** [comment]
Reasons for BookStack not being documented on a BookStack instance have been explained before here: https://github.com/BookStackApp/BookStack/issues/3386 

As to your other points:
- I'm not really sure what you're referring to with "soft versions". There is versioning for pages, you can link to versions (e.g. https://demo.bookstackapp.com/books/bookstack-demo-site/page/logging-in-to-the-demo-site/revisions/74) and set a change note when saving a page.
- Correct, content is currently not localizable, see https://github.com/BookStackApp/BookStack/issues/569 
- I'm not sure what should be "autoscreenshot", the BookStack content itself? Content to be documented in bookstack? There's loots of tools to take screenshots already and basically every OS and Browser has one included, I dont't think there's any need for an integrated solution.

**Comment by marcbria** [comment]
Thanks a lot for your answer Fabian.

As far I wanted to use BS to document a software and this soft will change between versions... by "soft versions" I mean that each page will need to refer the version of the software we are documenting. I was thinking in use tags, but I tested it and don't fits well with the idea.

In the other hand, by "autoscreenshot" I mean that, instead of adding an image, the tool automatically captures screenshots for certain urls. Actually I wrote it down because it is a requirement of our list but it is far away from the focus of BS, so forget the comment.

I will follow the rest of the threads closely.
